InnerEye/ML/configs/histo_configs/classification/BaseMIL.py

```diff
@@ -47,6 +49,10 @@ class BaseMIL(LightningContainer):
                                                   "dataset upfront and save it to disk.")
     # local_dataset (used as data module root_path) is declared in DatasetParams superclass
 
+    # slide dataset parameters:
+    slide_datatype: Optional[str] = param.String(default=None, allow_None=True, doc="Name of the slide dataset class if available.")
+    level: Optional[int] = param.Integer(doc="Downsampling level (e.g. 0, 1, 2) of the tiles if available.")
```

Comment:
```suggestion
    level: Optional[int] = param.Integer(doc="Downsampling level (e.g. 0, 1, 2) of the slides if available.")
```
we don't have a concept of level in our tiles dataset, at least for now

Reply:
shouldn't we set a default in this case?

Reply:
This is also removed from here. There is a default value in deepmil